pkg/apply/processor/create.go

```diff
@@ -89,7 +90,7 @@ func (c *CreateProcessor) PreProcess(cluster *v2.Cluster) error {
 	if err := MountClusterImages(cluster, c.Buildah); err != nil {
 		return err
 	}
-	runTime, err := runtime.NewDefaultRuntime(cluster, c.ClusterFile.GetKubeadmConfig())
+	runTime, err := kubernetes.NewDefaultRuntime(cluster, c.ClusterFile.GetKubeadmConfig())
```

Comment:
runtime.NewRuntime(runtimetype string,....)

Reply:
NewRuntime --> Should I put it in runtime or outside?